docs/install.md

```diff
@@ -0,0 +1,159 @@
+Installation
+---
+
+# Installing Spark Operator from Helm
+
+```bash
+export NAMESPACE=spark-operator
+
+helm repo add incubator http://storage.googleapis.com/kubernetes-charts-incubator
+helm install incubator/sparkoperator --namespace "${NAMESPACE}" --name spark-operator \
+     --set enableWebhook=true,sparkJobNamespace="${NAMESPACE}",enableMetrics=true
+     
+helm status "${SERVICE_NAME}"
+```
+
+Full list of configuration properties available at [Operator's Helm chart repo](https://github.com/helm/charts/tree/master/incubator/sparkoperator#configuration).
+
+After installing the Operator Helm Chart, regular `kubectl` commands can be used for checking the status of Operator:
+```bash
+kubectl config set-context --current --namespace=${NAMESPACE}
+kubectl get pods
+
+# example output:
+# NAME                                   READY   STATUS      RESTARTS   AGE
+# spark-sparkoperator-5c7bc7d487-wgw9z   1/1     Running     0          43m
+# spark-sparkoperator-init-hs7tr         0/1     Completed   0          43m
+
+# describe the operator pod
+kubectl describe pod spark-sparkoperator-5c7bc7d487-wgw9z
+
+# check operator logs
+kubectl logs spark-sparkoperator-5c7bc7d487-wgw9z
+``` 
+
+# Running Spark applications
+## Create Spark Driver Docker image
+Due to some issues in Spark Operator like [broken K8s client](https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/issues/591) 
+and the absence of Prometheus JavaAgent in the image a custom image needs to be built. An example image is available in  [examples/Dockerfile](examples/Dockerfile):
+```dockerfile
+FROM gcr.io/spark-operator/spark:v2.4.4
+
+# Setup for the Prometheus JMX exporter.
+RUN mkdir -p /etc/metrics/conf
+# Add the Prometheus JMX exporter Java agent jar for exposing metrics sent to the JmxSink to Prometheus.
+ADD https://repo1.maven.org/maven2/io/prometheus/jmx/jmx_prometheus_javaagent/0.11.0/jmx_prometheus_javaagent-0.11.0.jar /prometheus/
+COPY conf/metrics.properties /etc/metrics/conf
+COPY conf/prometheus.yaml /etc/metrics/conf
+```
+
+This custom image inherits from Spark image `v2.4.4` which has the fix for [broken K8s client issue](https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/issues/591).
+To build the image run:
+```bash
+# from docs/examples
+docker build -t [your image tag] .
+docker push [your image tag]
+```
+
+## Create service account for Driver
+An example RBAC configuration for service account `spark-dirver` is available in [examples/spark-driver-rbac.yaml](examples/spark-driver-rbac.yaml):
+```yaml
```

Comment:
copying was intentional here because the goal was both provide a doc as a whole and also have some working files which can be reused.  As now we have a [wiki page with Spark Operator user guide](https://github.com/mesosphere/kudo-spark-operator/pull/3), this whole   `install.md` is not needed in that shape. I'll update the PR so we have only spec files.